ocrd_network/ocrd_network/deployer.py

```diff
@@ -515,3 +518,39 @@ def start_native_processor_server(
         stdout.close()
         stdin.close()
         return re_search(r'xyz([0-9]+)xyz', output).group(1)  # type: ignore
+
+    # TODO: No support for TCP version yet
+    def start_unix_mets_server(self, mets_path: str) -> str:
+        mets_server_url = f'/tmp/{safe_filename(mets_path)}'
+        if Path.exists(Path(mets_server_url)):
```

Comment:
Perhaps try a request to `/workspace_name` to test whether it is running, that works for both UDS and HTTP server.

Reply:
There is `/workspace_path` that returns `workspace.directory` as content. Should we implement another endpoint just for checking the running status of the server that returns just a status code of 200?